README.md

```diff
@@ -1 +1,125 @@
-# Hello-MV-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+ - [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+> This is my first MV Project as a Micronaut. In this project, I created a branch and applied linters.
+
+## 🛠 Built With <a name="built-with"></a>
+* HTML 
+* CSS
+### Tech Stack <a name="tech-stack"></a>
+> Hello Microverse!
+> ### Key Features <a name="key-features"></a>
+
+- **Using linter**
+- **Using gitHub flow**
+- **Creating a branch**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Here is my first project as a Micronaut to understand setting up linter.
+
+### Prerequisites
+
+In order to run this project you need: to download it to your desired PC.
+
+### Setup
+
+Clone this repository to your desired folder:
+1. Click Code button (shown in green color)
+2. Copy URL.
+3. Please clone it with the following command: git clone git@github.com:myaccount/my-project.git
+
```

Comment:
- [ ] You have excluded many sections under getting started and this has affected the professionalism of your readme. Please do not delete sections of this template. The only optional section is the **FAQs**. You can also from time to time leave out the **tests & deployment** if your project does not have tests & is not deployed 👍🏾 
![Screenshot from 2023-05-30 09-16-53](https://github.com/mryeminthu/Hello-MV-Project/assets/99656687/5f472621-1c20-4c89-bb7b-076a9760b993)